lib/manager/bundler/readme.md

```diff
@@ -4,39 +4,29 @@
 
 #### Name of package manager
 
-[Bundler](https://bundler.io/)
-
----
+Bundler
 
 #### What language does this support?
 
 Ruby
 
----
-
 #### Does that language have other (competing?) package managers?
 
-No mainstream
+No
 
 ## Package File Detection
 
 #### What type of package files and names does it use?
 
-"Gemfiles", usually named `Gemfile`.
-
----
+Package - gem. Name - Gemfile or gemspec
 
 #### What [fileMatch](https://renovatebot.com/docs/configuration-options/#filematch) pattern(s) should be used?
 
-`["(^|/)Gemfile$]`
-
----
+`/Gemfile|.gemspec/`
 
 #### Is it likely that many users would need to extend this pattern for custom file names?
 
-It's possible, but not a common case.
-
----
+No
```

Comment:
This indicates that the above should be an exact match filename rather than partial match (regex needs updating)